src/gigs-board/entity/github-repo/board.jsx

```diff
@@ -0,0 +1,234 @@
+        <h5 className="h5 d-inline-flex gap-2 m-0">
+          <i className="bi bi-kanban-fill" />
+          <span>{title}</span>
+        </h5>
+
+        <p className="m-0 py-1 text-secondary text-center">{description}</p>
+      </div>
+
+      <div className="d-flex justify-content-end gap-3">
+        {pageURL ? (
+          <a
+            className="card-link d-inline-flex me-auto"
+            href={pageURL}
+            rel="noreferrer"
+            role="button"
+            target="_blank"
+            title="Link to this board"
+          >
+            <span className="hstack gap-2">
+              <i className="bi bi-share" />
+              <span>Open in new tab</span>
+            </span>
+          </a>
+        ) : null}
+
+        {pageURL ? (
+          <button
+            className="btn btn-sm btn-outline-secondary d-inline-flex gap-2"
+            onClick={() => clipboard.writeText(pageURL)}
+          >
+            <i className="bi bi-clipboard-fill" />
+            <span>Copy link</span>
+          </button>
+        ) : null}
+
+        {isEditable ? (
+          <button
+            className="btn btn-sm btn-primary d-inline-flex gap-2"
+            onClick={editorTrigger}
+          >
+            <i className="bi bi-wrench-adjustable-circle-fill" />
+            <span>Configure</span>
+          </button>
+        ) : null}
+      </div>
+
+      <div className="d-flex gap-3" style={{ overflowX: "auto" }}>
+        {Object.keys(columns).length > 0 ? (
+          Object.values(columns).map((column) => (
+            <div className="col-3" key={column.id}>
+              <div className="card">
+                <div
+                  className={[
+                    "card-body d-flex flex-column gap-3",
+                    "border border-2 border-secondary rounded-2",
+                  ].join(" ")}
+                >
+                  <h6 className="card-title h6 d-flex align-items-center gap-2 m-0">
+                    {column.title}
+
+                    <span className="badge rounded-pill bg-secondary">
+                      {(state.ticketsByColumn[column.id] ?? []).length}
+                    </span>
+                  </h6>
+
+                  <p class="text-secondary m-0">{column.description}</p>
+
+                  <div class="d-flex flex-column gap-3">
+                    {(state.ticketsByColumn[column.id] ?? []).map((data) =>
+                      widget(
+                        "entity.github-repo.ticket",
+                        { data, format: "card" },
+                        data.id
+                      )
+                    )}
+                  </div>
+                </div>
+              </div>
+            </div>
+          ))
+        ) : (
+          <div
+            className={[
+              "d-flex align-items-center justify-content-center",
+              "w-100 text-black-50 opacity-50",
+            ].join(" ")}
```

Comment:
Is this because of fmt?

Reply:
No, I do this either when some part of className is dynamic or when it's too big ( wider that Prettier' default 80 chars ) because it's simply harder to work on layouting otherwise